src/components/Popup/Popup.tsx

```diff
@@ -35,8 +35,10 @@ export class Popup extends BaseComponent<IPopupProps, {}> {
       this._originalFocusedElement &&
       this._containsFocus &&
       this._originalFocusedElement as any !== window) {
-
-      this._originalFocusedElement.focus();
+      // This slight delay is required so that we can unwind the stack, let react try to mess with focus, and then
```

Comment:
hmmm shouldn't the originalFocusedElement be set in componentWillMount instead of DidMount?